x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ClassificationIT.java

```diff
@@ -454,7 +459,7 @@ public void testTwoJobsWithSameRandomizeSeedUseSameTrainingSet() throws Exceptio
         String sourceIndex = "classification_two_jobs_with_same_randomize_seed_source";
         String dependentVariable = KEYWORD_FIELD;
 
-        createIndex(sourceIndex);
+        createIndex(sourceIndex, true);
```

Comment:
I think it'd be nice to have a new test like you added in `DatafeedJobsID`. Having a test that explains it works on data streams in its name adds to the documentation of what those jobs should support. Also, if that test failed but all others worked fine, we'd immediately know it's because of data streams and not something else.